packages/server/src/api/controllers/deploy/index.js

```diff
@@ -0,0 +1,51 @@
+const CouchDB = require("pouchdb")
+const PouchDB = require("../../../db")
+const { uploadAppAssets } = require("./aws")
+
+function replicate(local, remote) {
+  return new Promise((resolve, reject) => {
+    const replication = local.sync(remote)
+
+    replication.on("complete", () => resolve())
+    replication.on("error", err => reject(err))
+  })
+}
+
+async function replicateCouch(instanceId, clientId) {
+  const databases = [`client_${clientId}`, "client_app_lookup", instanceId]
+
+  const replications = databases.map(local => {
+    const localDb = new PouchDB(local)
+    const remoteDb = new CouchDB(
+      `${process.env.DEPLOYMENT_COUCH_DB_URL}/${local}`
```

Comment:
I had expected a remote client db to be called something like `client_abcd-1234-...` , whereas the local client db is going to be `client_1` - is this not likely to cause a clash ?

Reply:
Also, is there currently anything stoping people modifyng the code, and replicating to an instance that is not theirs ? (if they knew someones instance id )

Probably fine for beta

Reply:
oh - does everyone get their own unique DEPLOYMENT_COUCH_DB_URL ? is this how this works ?

Reply:
Nope, this is changing too. Each account will have a couch user, and they get their credentials back from AWS. The client DB thing is valid though - should we not just make it `client_<uuid>`?